src/app/pages/wallets/management/pages-wallets-management.component.ts

```diff
@@ -31,14 +25,13 @@ export class PagesWalletsManagementComponent implements OnInit {
       private readonly myWalletsService: PagesWalletsManagementService,
       private readonly confirmDialogService: ConfirmDialogService,
       private readonly loadingDialogService: LoadingSnackbarService,
-      private readonly systemNotificationsService: SystemNotificationsService,
-      private readonly dialog: MatDialog,
+      private readonly pagesWalletsManagementEditorService: PagesWalletsManagementEditorService,
```

Comment:
Fo easier use you can call it just `editorService`, there is very small chance you will use here more than one `editorService`